Website/engine.js

```diff
@@ -0,0 +1,202 @@
+function engine(data) {
+    $("#top").empty();
+    $("#main").empty();
+    $("#top").hide();
+    $("#main").hide();
+    let ab = {"scroll": false};
+    let ao = {"scroll": false};
+    if (data.type === "overview") {
+        ao = overview(data.data);
+    } else if (data.type === "personalised") {
+        ab = personalised(data.data);
+    }
+
+    $("#main").children().each(function () { 
+        $(this).hide();
+    });
+
+    $("#main").show();
+    $("#top").fadeIn("slow");
+
+    let eT = 0;
+    $('#main').children().each(function () { 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8404081)
```suggestion
    $("#main").children().each(function () { 
```